src/components/Workspace.jsx

```diff
@@ -210,6 +211,10 @@ class Workspace extends React.Component {
   }
 
   _renderOutput() {
+    if (includes(this.props.ui.minimizedComponents, 'preview')) {
```

Comment:
Sure. I was just following the previous naming convention where the output component was named `'preview'`